### PR TITLE
Math Magician: Setup a New React App

### PR DESCRIPTION
**Pull request by anyars-encarta**
In this project:

- [ ] I installed [Node.js](https://nodejs.org/en/) on my machine.
- [ ] I created a React Application using [Create React App](https://create-react-app.dev/).
- [ ] The application runs in the browser on `localhost:3000` without errors.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+          className="App-link"
+          href="https://reactjs.org"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Learn React
+        </a>
+      </header>
+    </div>
+  );
```
Comment (V-Blaze) [inline src/App.tsx]: - [ ] You have done great so far. :+1:  But there are some issues you need to fix before moving to the next milestone. :smile: :small_airplane: 
- [ ] Looking at the video below, you can see that when I start the app, the browser is black without displaying the default react logo and text. I'll suggest you look into this as your app is meant to run normally after the setup. :+1: :100: 
- [ ] Also it seems you are using typescript in this project. this is not part of the requirement, I'll suggest you revert your code to using plane vanilla JS. :+1: 

- - Video reference:

https://github.com/anyars-encarta/math-magician/assets/107632780/2f01ac59-be30-4919-af55-3de013877e58

File: README.md [hunk 2]
```diff
@@ -1 +1,180 @@
...
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- INTRO -->
+# 📖 React App <a name="about-project"></a>
+
```
Comment (V-Blaze) [inline README.md]: - [ ] As required by MIcroverse, your readme file needs to be descriptive and look professional, I'll suggest you remove this react logo and also name your project properly. Also, this is a react app, but the project name is `Math Magician`. :+1: :+1: 

- - Screenshot:
Selection_006

File: README.md [hunk 3]
```diff
@@ -1 +1,180 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> To get a local copy up and running, follow these steps.
+
+<!-- SETUP -->
+### Setup
```
Comment (V-Blaze) [inline README.md]: - [ ] I'll suggest you remove the live demo section as your project hasn't been deployed yet or you can add the text `Comming Soon`. Also, I'll suggest you add the link to clone your repository in the getting local copy section. :+1: :smile: :wave: 


- - Screenshot:
Selection_007

File: README.md [hunk 4]
```diff
@@ -1 +1,180 @@
...
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/anyars-yussif-1a179769/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **Add a calculator feature**
```
Comment (V-Blaze) [inline README.md]: - [ ] I'll suggest you remove these as they are placeholder comments which help you fill out your readme. :small_airplane: 

- - Screenshot:
Selection_008

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @anyars-encarta 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Correct Git flow✔
-  Linters setup correctly 👋 
-  Meaningful commit messages✔

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.tsx (hunk 1 above)
Comment (anyars-encarta) [reply src/App.tsx]: Looking into it

File: README.md (hunk 2 above)
Comment (anyars-encarta) [reply README.md]: I will correct that

File: README.md (hunk 4 above)
Comment (anyars-encarta) [reply README.md]: Noted

File: README.md [hunk 5]
```diff
@@ -1 +1,185 @@
...
+You can deploy this project using:
+>1.  GitHub Pages
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
```
Comment (DammyShittu) [inline README.md]: - [ ] Kindly remove this section description as requested by the previous reviewer.

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @anyars-encarta,

## Highlights
✔️ App starts with required command

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 5 above)
Comment (anyars-encarta) [reply README.md]: Sorry for the oversight sir

**Review by usorfaitheloho — APPROVED** [review]
Hi @anyars-encarta  , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**